.gitignore

```diff
@@ -47,8 +47,8 @@ node_modules/
 *.tgz
 
 # dotenv environment variables file
-.env
-.env.test
+.env*
+!.env.local.example
```

Comment:
Looks good to me :) 

Is there any possibility of others wanting to add an env file with some other name pattern? Eg. `.env.example`?

Reply:
I don't think so: we only use `.env` files in the context of Next.js which wants us to use a `.env.local` for our secrets.

So as far as I can see, there is no need for a `.env.example` right now.

Reply:
There is a chance, that this might happen at some point. The solution for me would be for that person to add another line `!.env.example` which would be an explicit and secure way in my opinion.